app/controllers/application_controller.rb

```diff
@@ -74,4 +74,39 @@ def set_no_cache_header
   def touch_current_user
     current_user.touch
   end
+
+  # The following is from the now deprecated fastly-rails gem
+  # https://github.com/fastly/fastly-rails/tree/master/lib/fastly-rails/action_controller
+
+  # Sets Cache-Control and Surrogate-Control HTTP headers
+  # Surrogate-Control is stripped at the cache, Cache-Control persists (in case of other caches in front of fastly)
+  # Defaults are:
+  #  Cache-Control: 'public, no-cache'
+  #  Surrogate-Control: 'max-age: 30 days
+  # custom config example:
+  #  {cache_control: 'public, no-cache, maxage=xyz', surrogate_control: 'max-age: blah'}
+  def set_cache_control_headers(max_age = 1.day.to_i, opts = {})
```

Comment:
`1.day.to_i` used to be held in `FastlyRails.configuration`, but we'll be getting rid of that soon. I thought about moving it to an ENV variable, but I think that might be overkill. It shouldn't change by environment and it would create more ENV variables that other new communities may find noise (i.e. they don't set up Fastly). I'm open to moving this somewhere else if someone has a suggestion!